src/client/ducks/pullrequests/selectors/index.js

```diff
@@ -55,11 +55,11 @@ export const getPullRequest = createSelector(
   })
 
 // Sometimes we don't need Denormalization of users, when we need just issues or comments
-export const getPullRequestNormlized = createSelector(
+export const getPullRequestNormalized = createSelector(
   getPullRequestsEntities, getPullRequestId,
   (entities: Object, id: string): Object =>
-    (id ? entities[id] : {})
- )
+    (id ? entities[id] || {} : {})
```

Comment:
@kkateq this fixes the reloading page problem that I was just trying to fix, I was just trying this as well. I guess you are adding a unit test as you suggested? same potential problem happens in line 52 (above)